test/signin.js

```diff
@@ -0,0 +1,30 @@
+let app = require('../app');
+let chai = require('chai');
+let request = require('supertest')
+let should = chai.should();
+request = request('http://localhost:3000')
```

Comment:
Actually, just thought about it, you should be doing something like:

```
request(app)
.post(...)
```

So you don't really need the `request = request('http://localhost:3000')`